README.md

```diff
@@ -499,6 +488,41 @@ getBoundingBox( target : Box3 ) : Box3
 
 Get the bounding box of the geometry computed from the root node bounds of the BVH. Significantly faster than `BufferGeometry.computeBoundingBox`.
 
+### getTriangleHitPointInfo
+
+```js
+getTriangleHitPointInfo(
+	point: Vector3,
+	geometry : BufferGeometry,
+	triangleIndex: Number
+	target: Object
+): Object
+```
+
+This function returns information of a point related to a geometry. It returns the `target` object or a new one if passed `undefined`:
+
+```js
+target : {
+	point: Vector3,
+	face: {
+		a: Number,
+		b: Number,
+		c: Number,
+		materialIndex: Number,
+		normal: Vector3
+	},
+	uv: Vector2
+}
+```
+
+- `point`: The same point
+- `a`, `b`, `c`: Triangle indices
+- `materialIndex`: Face material index or 0 if not available.
```

Comment:
From the function implementation it looks like the materialIndex is always set to 0, right? Technically the material index should be based on the `geometry.groups` list, but a better approach will require changes to the "shapecast" function so the current group id is passed down to the callbacks, which I'll make an issue for.

Reply:
#325